CHANGELOG.md

```diff
@@ -15,7 +15,6 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 - IsNullable flag for Field (#302)
 - More linters on CI (#310)
 - Meaningful description for read/write socket errors (#129)
-- Support password and password file to decrypt private SSL key file (#319)
```

Comment:
Wait, shouldn't there also be "move SSL support to a separate module"?

Reply:
Added the `CHANGELOG` entry.